### ISSUE TITLE
NTR: Bacteria in peritoneal fluid (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7915]
New term request Bacteria in peritoneal fluid (nadea3_200115185424):   UMLS: 5154140

**Comment by pnrobinson** [comment]
@MickeySegal
is this a microscopic or culture finding?

**Comment by MickeySegal** [comment]
It seems that cultures are preferred:
https://www.news-medical.net/health/Peritonitis-Diagnosis.aspx
 A polymorphonuclear (PMN) cell count that is greater than 250 cells per mm3 and the yield of cultures of the ascitic fluid represent the gold standard in diagnosing this condition.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000954
- label: Positive peritoneal fluid bacterial culture
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Growth of bacteria in a culture dervied from ascites fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7915
- synonyms:
